### PR TITLE
initial new service with updated SQL

### PR DESCRIPTION
**Pull request by aaronhans**
We are modifying the SQL used to retrieve Vaccines delivered by HPI both for the partially/fully vaccinated people chart and the total doses chart shown on the /vaccines page.

This new SQL uses improved views with cleaner data provided by CDPH.

The output json uses a different field structure which is clearer because we are no longer referring to first_dose which doesn't make sense with the single dose vaccine from J&J.

Because the output json is different and we want to keep it that way this new version of the service writes to a v2 file location.

Jim has updated the charts in staging to refer to this new file location in the staging branch of covid-static.

The production charts should be modified to use the new file location on files.covid19

### CODE REVIEW

**Comment by aaronhans** [comment]
We are coordinating with CDPH to get this data reviewed today. We should modify this service to take advantage of Chach's documentation upgrades ASAP

**Comment by aaronhans** [comment]
PRs created by running this locally:
https://github.com/cagov/covid-static/pull/363
https://github.com/cagov/covid-static/pull/362

**Review by chachasikes — APPROVED** [review]
This is awesome! Thank you for adding docs too! 
I'll pull this and line it up with what I did yesterday in my working branch.

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,3406 @@
 {
   "name": "cagov-cron",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```
Comment (carterm) [inline package-lock.json:6]: Whoa.  Lots of new packages.  What's going on here?

Comment (chachasikes) [reply package-lock.json:6]: npm has an issue we need to discuss as a team (and decide on what upgrades we want to do with our npm packges) & then we can summarize what's going on - it has to do with npm versions & needing to upgrade to the new version of npm

Comment (aaronhans) [reply package-lock.json]: I grabbed package-lock from master because I didn't intend to update packages here

**Review by carterm — APPROVED** [review]
Looks like a fine new service.

**Pull request merged**